Measurements/Tests.qs

```diff
@@ -349,6 +349,17 @@ namespace Quantum.Kata.Measurements {
 
         CheckSuperpositionBitstringsOneMeasurement(5, [30,14,10,6],     // [11110,01110,01010,00110]
                                                       [1,17,21,25]);    // [00001,10001,10101,11001]
+        CheckSuperpositionBitstringsMeasurement(2, [1,2],       // [01,10]
```

Comment:
Please note that this task should use `CheckSuperpositionBitstringsOneMeasurement` operation for testing each individual test case instead of `CheckSuperpositionBitstringsMeasurement` used by the next task.

Reply:
done